causing/graph.py

```diff
@@ -231,9 +231,17 @@ def specific_color_str(wei: float, line_colored: bool) -> str:
         show_in_percent,
         node_name,
     )
-    dot_str = "digraph { \n" + form + x_dot + y_dot + "        }"
 
-    dot_str = graphviz.unflatten(dot_str, stagger=3)
+    if invisible_node:
+        invis_node = (
+            '         "{}" -> "{}" [style = "invisible", arrowhead="none"];\n'.format(
+                invisible_node[0][0], invisible_node[0][1]
+            )
+        )
+        dot_str = "digraph { \n" + form + x_dot + y_dot + invis_node + "        }"
+    else:
+        dot_str = "digraph { \n" + form + x_dot + y_dot + "        }"
```

Comment:
No need to duplicate the `dot_str = ...` line.
```suggestion
    else:
        inivis_node = ''
    dot_str = "digraph { \n" + form + x_dot + y_dot + invis_node + "        }"
```